# this patch is currently not used, not providing entry points for all interpreters

Index: b/setup.py
===================================================================
--- a/setup.py
+++ b/setup.py
@@ -38,8 +38,17 @@ def _gen_console_scripts():
     )
     if any(os.environ.get(var) not in (None, "", "0") for var in var_names):
         return
-    yield ("easy_install-{shortver} = setuptools.command.easy_install:main"
-        .format(shortver=sys.version[:3]))
+    if sys.version_info[0] < 3 and 'XPYVERSISONS' in os.environ:
+        for version in os.environ['XPYVERSIONS'].split():
+            yield ("easy_install-{shortver} = setuptools.command.easy_install:main"
+                   .format(shortver=version))
+    elif sys.version_info[0] >= 3 and 'XPY3VERSIONS' in os.environ:
+        for version in os.environ['XPY3VERSIONS'].split():
+            yield ("easy_install-{shortver} = setuptools.command.easy_install:main"
+                   .format(shortver=version))
+    else:
+        yield ("easy_install-{shortver} = setuptools.command.easy_install:main"
+               .format(shortver=sys.version[:3]))
 
 console_scripts = list(_gen_console_scripts())
 
